client/src/service/deal_cards.ts

```diff
@@ -0,0 +1,30 @@
+function dealCards(players: number, rounds: number, pack: number[] = []) {
```

Comment:
There might be some overlap / conflict with this method and the Game service for initialiseGame. 

Do you want to see if you can hook dealCards into initaliseGame function?  I think it may change that or this method implementation. 

Happy to pair on Monday if easier?

Reply:
Not sure I understand the hooking up - I could call it from initialiseGame()?  

Or do you mean cut the code and paste into intialiseGame()?

Alternatively, I'm happy to drop it from the PR.  I just thought I was going to use it in a test, then wrote a test for it to get my head into a "testing" space.